packages/@holaplexui-react/src/components/Modal.tsx

```diff
@@ -59,22 +59,21 @@ export function Modal(props: ModalProps) {
             >
               <Dialog.Panel
                 className={clsx(
-                  'z-40 w-full max-w-md transform !overflow-visible rounded-2xl bg-gray-900  text-left align-middle transition-all',
-                  'scrollbar-thumb-rounded-full relative flex h-full max-h-screen w-full flex-col overflow-y-auto rounded-xl bg-gray-900  text-white shadow-md scrollbar-thin scrollbar-track-gray-900  sm:h-auto  sm:max-w-lg',
+                  'relative z-40 flex flex-col bg-white h-full transform !overflow-visible align-middle transition-all scrollbar-thumb-rounded-full overflow-y-auto rounded-md shadow-md scrollbar-thin max-h-screen max-w-md sm:h-auto sm:max-w-lg',
```

Comment:
All the styling including border radius shadow bg color should be set by the `modal-content` class